libgnucash/engine/Account.cpp

```diff
@@ -5665,38 +5667,36 @@ build_non_bayes (const char *key, const GValue *value, gpointer user_data)
     g_free (guid_string);
 }
 
-static std::tuple<std::string, std::string, std::string>
+static std::tuple<std::string, std::string>
```

Comment:
This function is expensively silly. Do it inline in `build_bayes` and use pointer math on suffix instead of constructing a string just to use `substr()`. The only string construction should be `account_guid` because `gnc::GUID::from_string()` takes a `const std::string&`.

Reply:
Done